src/pipeline_v3/docs/CI_DATABASE_TESTING_GUIDE.md

```
@@ -0,0 +1,340 @@
+
+manager = TestDatabaseManager()
+success, tenant_info = manager.ensure_databases_ready()
+if success:
+    tenant_id = tenant_info['tenant_id']
+    # Run tests
+    manager.cleanup_test_tenant()
+```
+
+## Environment Variables
+
+### Local Development
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)